confirmation/models.py

```diff
@@ -73,7 +73,17 @@ def create_confirmation_link(obj: ContentType,
     elif isinstance(obj, Realm):
         realm = obj
 
-    Confirmation.objects.create(content_object=obj, date_sent=timezone_now(), confirmation_key=key,
+    validity_in_days = settings.CONFIRMATION_LINK_DEFAULT_VALIDITY_DAYS
+
+    if (confirmation_type is Confirmation.INVITATION or
+            confirmation_type is Confirmation.MULTIUSE_INVITE):
+        validity_in_days = obj.invite_expires_in
```

Comment:
Hmm, this feels like we're duplicating an existing value.

@mateuszmandera how do you think we should evolve this detail?  Is the right model to write out migration to move this field out of PreregistrationUser and into Confirmation?  Or is that problematic, because we need to actually access the field given a PreregistrationUser object (One thought is that perhaps we should add a PreregistrationUser => Confirmation foreign key to make that access path convenient?).

Reply:
It seems that @shanukun resolves this with `confirmation = GenericRelation("confirmation.Confirmation", related_query_name="prereg_user")`, which looks the right way of enabling reverse lookups in a situation like this https://docs.djangoproject.com/en/3.1/ref/contrib/contenttypes/#reverse-generic-relations